src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import java.util.*;
+
+public class UtilsTest {
+	
+	private Utils utils = new Utils();
+	
+	@Test public void isNullTest() {
+		Object object = null;
+		Boolean value = utils.isAnyNull(object);
+		assertTrue(value.equals(true));
+	}
+	
+	@Test public void isNotNullTest() {
```

Comment:
It is okay to have `@Test` and method's signature in the same line. But at most cases, as well as to keep consistent convention with `ParserTest.java`, you'd better to place `@Test` at the line before method's signature.